stream/src/main/scala/org/apache/pekko/stream/impl/ActorMaterializerImpl.scala

```diff
@@ -195,10 +197,14 @@ private[pekko] class SubFusingActorMaterializerImpl(
  * INTERNAL API
  */
 @InternalApi private[pekko] object StreamSupervisor {
-  def props(attributes: Attributes, haveShutDown: AtomicBoolean): Props =
+  def props(attributes: Attributes, haveShutDown: AtomicBoolean): Props = {
+    val dispatcher = attributes.mandatoryAttribute[ActorAttributes.Dispatcher].dispatcher
     Props(new StreamSupervisor(haveShutDown))
       .withDeploy(Deploy.local)
-      .withDispatcher(attributes.mandatoryAttribute[ActorAttributes.Dispatcher].dispatcher)
+      .withDispatcher(dispatcher)
+      .withMailbox(PhasedFusingActorMaterializer.MailboxConfigName)
```

Comment:
Here, it seems that the dispatcher can rewrite through attributes.
Is mailbox necessary? I am not sure.

Reply:
This code is right dude.